discovery-provider/alembic/versions/4c3784d41d41_add_ip_to_route_metrics.py

```diff
@@ -0,0 +1,52 @@
+"""Add IP to route metrics
+
+Revision ID: 4c3784d41d41
+Revises: 277bad300c09
+Create Date: 2020-08-28 17:21:43.308374
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4c3784d41d41'
+down_revision = '277bad300c09'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    # Route metrics
+    op.drop_constraint('route_metrics_pkey', 'route_metrics')
+    op.execute("ALTER TABLE route_metrics ADD COLUMN id SERIAL NOT NULL PRIMARY KEY")
```

Comment:
i believe this is how `Serial` works, it adds a new auto incrementing value to everything there. I'll run against a prod db to double check

Reply:
oh interesting. i didn't know it would do that retroactively. if you could confirm that would be great. also i saw this which says use identity columns instead of serial https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial

Reply:
nice find :)